src/sparseml/pytorch/models/registry.py

```diff
@@ -91,23 +92,33 @@ def create(
             loaded in model, False otherwise; default True
         :param ignore_error_tensors: tensors to ignore if there are errors in loading
         :param kwargs: any keyword args to supply to the model constructor
-        :return: the instantiated model
+        :return: The instantiated model if key is given else a Tuple containing
+            the instantiated model and the loaded key
         """
-        if key not in ModelRegistry._CONSTRUCTORS:
+        _key = key
```

Comment:
why add the underscore to the name?

Reply:
removed for a more generic name `key_copy`

Reply:
Fixed for a more generic name